flake8_django.py

```diff
@@ -53,6 +40,35 @@ def visit_Call(self, node):
                     )
                 )
 
+    def capture_url_issues(self, node):
+        for keyword in node.keywords:
+            if keyword.arg == 'name' and '-' in keyword.value.s:
+                self.issues.append(
+                    DJ03(
+                        lineno=node.lineno,
+                        col=node.col_offset,
+                        parameters={}
+                    )
+                )
+                return
+
+    def visit_Call(self, node):
+        """
+        blank=True is not recommended to be used in fields specified in NOT_BLANK_TRUE_FIELDS.
+
+        null=True is not recommended to be used in fields specified in NOT_NULL_TRUE_FIELDS
+        """
+        if isinstance(node.func, ast.Attribute):
+            call = node.func.attr
+        else:
+            call = node.func.id
+
+        if call == 'url':
+            self.capture_url_issues(node)
+
+        if call in NOT_NULL_TRUE_FIELDS or call in NOT_BLANK_TRUE_FIELDS:
+            self.capture_field_issues(node, call)
```

Comment:
What about adding these pre-conditions inside the `capture_field_issues` function, given that this can be used to capture other kinds of field issues in the future?

Reply:
👍 sounds good!